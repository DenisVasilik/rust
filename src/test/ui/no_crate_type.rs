// regression test for issue 11256
#![crate_type]  //~ ERROR attribute must be of the form `#[crate_type = "..."]`

fn main() {
    return
}
