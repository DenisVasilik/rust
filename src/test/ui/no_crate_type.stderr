error: attribute must be of the form `#[crate_type = "..."]`
  --> $DIR/no_crate_type.rs:2:1
   |
LL | #![crate_type]  //~ ERROR attribute must be of the form `#[crate_type = "..."]`
   | ^^^^^^^^^^^^^^

error: aborting due to previous error

